source/operator.rst

```diff
@@ -171,546 +317,707 @@
 +-----------------+-------------------------------------------------------------------------------------------+
 |                                                                                                             |
 +-----------------+-------------------------------------------------------------------------------------------+
-|                 | .. _spec-backrests3verifytls:                                                             |
+|                 | .. _backup-storages-type:                                                                 |
 |                 |                                                                                           |
-| **Key**         | `backrestS3VerifyTLS <operator.html#spec-backrests3verifytls>`_                           |
+| **Key**         | `backup.storages.<storage-name>.type <operator.html#backup-storages-type>`_               |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Value**       | boolean                                                                                   |
+| **Value**       | string                                                                                    |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Example**     | ``false``                                                                                 |
+| **Example**     | ``s3``                                                                                    |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Description** | Enables or disables TLS verification for pgBackRest                                       |
+| **Description** | The cloud storage type used for backups. Only ``s3`` type is supported                    |
 +-----------------+-------------------------------------------------------------------------------------------+
 |                                                                                                             |
 +-----------------+-------------------------------------------------------------------------------------------+
-|                 | .. _spec-shutdown:                                                                        |
+|                 | .. _backup-storages-s3-endpointurl:                                                       |
 |                 |                                                                                           |
-| **Key**         | `shutdown <operator.html#spec-shutdown>`_                                                 |
+| **Key**         | `backup.storages.s3.<storage-name>.endpointURL                                            |
+|                 | <operator.html#backup-storages-s3-endpointurl>`_                                          |
```

Comment:
The key is `backup.storages.<storage_name>` , not `backup.storages.s3.<storage-name>`

same issue below in the doc.